src/node_quic_socket.cc

```diff
@@ -656,7 +658,9 @@ BaseObjectPtr<QuicSession> QuicSocket::AcceptInitialPacket(
   switch (QuicServerSession::Accept(&hd, data, nread)) {
     case QuicServerSession::InitialPacketResult::PACKET_VERSION:
       SendVersionNegotiation(version, dcid, scid, addr);
-      // Fall-through to ignore packet
+      // Return explicitly here rather than falling-through to
+      // avoid compiler warning
+      return {};
```

Comment:
I think we just need to use a fall through comment format that the compiler understands, `// Fall through` or something might work :)